src/commands/list.rs

```diff
@@ -46,8 +46,8 @@ pub fn list(matches: &ArgMatches<'_>, api: &mut API) -> Result<(), Box<dyn std::
       None => return Err(Box::new(APIError::BadFormat)),
     };
     let subject_line = format!("{} ({})", display_name, name);
-    println!("{}", &subject_line);
-    println!("{}", "=".repeat(subject_line.len()));
+    println!("\n\x1B[36m{}\x1B[0m", &subject_line);
```

Comment:
Maybe these should be constants? Should also only do this on a `TERM` which supports colors. Maybe there's a crate we can use?

Reply:
I think the two options should be either saving them as vars to reference or using the term crate

Reply:
I feel like less external crates the better but up to you